include/pfs/drp/stella/fitLine.h

```diff
@@ -0,0 +1,46 @@
+#include "pfs/drp/stella/Spectrum.h"
+
+namespace pfs {
+namespace drp {
+namespace stella {
+
+
+struct FitLineResult {
+    double rms;  ///< RMS of the residuals
+    bool isValid;  ///< Did the fit succeed?
+    double amplitude;  ///< Amplitude of the Gaussian
+    double center;  ///< Center of the Gaussian (pixels)
+    double width;  ///< Width (standard deviation, sigma) of the Gaussian (pixels)
+    double bg0;  ///< Background intercept
+    double bg1;  ///< Background slope
+    std::size_t num;  ///< Number of pixels
+
+    FitLineResult(double _rms, bool _isValid, double _amplitude, double _center, double _width,
+                  double _bg0, double _bg1, std::size_t _num
+        ) : rms(_rms), isValid(_isValid), amplitude(_amplitude), center(_center), width(_width),
+            bg0(_bg0), bg1(_bg1), num(_num) {}
+};
+
+
+/** Fit a line to the spectrum
+ *
+ * We fit a Gaussian line plus linear continuum to a nominated portion of the
+ * spectrum. The linear continuum is expressed as bg0 + bg1*(x - center).
+ *
+ * @param spectrum : Spectrum to fit.
+ * @param peak : estimated position of line to fit (pixels).
+ * @param width : estimated width (standard deviation, sigma) of line to fit (pixels).
```

Comment:
Would people prefer a FWHM?  Whatever we use, please change the name to be more precise (e.g. `rmsSize`; `FWHM`).